### DISCUSSION TITLE
What is the reference point of V in the output parameters of TMS simulation?

### DISCUSSION THREAD

**Discussion by yyx1027** [simnibs/simnibs#306] [9. Results and analysis]
Dear SimNIBS experts,

I want to compute the cortical midpotential in TMS simulation post-processing. But I don't know what the reference point of V is in the simulation results. And whether we can use E to calculate the cortical potential, and how we can calculate it.

Can you suggest how to solve it?

Thank you, greetings,

![image](https://github.com/simnibs/simnibs/assets/138636440/365bd529-73d7-4301-a5d2-a4c5f65ff1e1)

**Comment by TorgeW** [comment]
The reference point is an automatically selected node in the lower part of the head.
What do you mean by cortical midpotential? The potential is only defined up to a constant.

**Reply by yyx1027** [reply]
Thank you very much for your answer！

I actually wanted to use the simulated electric field results to calculate the potential at any position in the cortex as the extracellular potential of the neuron, and to study the effect of the electric field generated by the TMS coil on the neuron.

There is confusion about how to convert the induced electric field into the calculation of the extracellular potential, and it is hoped that experts can give a solution.

Thank you,

**Reply by TorgeW** [reply]
For TMS you cannot use the potential but rather the pseudo potential which has to be calculated from the  e field on each neuronal cell by integrate along the arbors.

That is because the TMS induced E field is not curl free.
